go.mod

```
@@ -1,6 +1,6 @@
 module github.com/onflow/cadence
 
-go 1.18
+go 1.19
```

This indicates the minimum version of Go. We should be able to keep requiring Go 1.18

```suggestion
go 1.18
```

---

wouldn't that create issues for contributors since they'll fail ci-lint?